iotfunctions/db.py

```diff
@@ -461,28 +441,6 @@ def __init__(self, credentials=None, start_session=False, echo=False, tenant_id=
                                          'DATABASE=xxx;HOSTNAME=xxx;PORT=xxx;UID=xxx;PWD=xxx[;SECURITY=xxx]' % connection_string_from_env)
                     self.db_type = 'db2'
 
-                elif db_type_from_env == 'POSTGRESQL':
-                    sqlalchemy_connection_string = 'postgresql+psycopg2://' + connection_string_from_env
-                    native_connection_string = connection_string_from_env
-                    try:
-                        # Split connection string according to user:password@hostname:port/database
-                        first, last = connection_string_from_env.split("@")
-                        uid, pwd = first.split(":", 1)
-                        hostname_port, database = last.split("/", 1)
-                        hostname, port = hostname_port.split(":", 1)
-                        self.credentials['postgresql'] = {"username": uid, "password": pwd, "db": database,
-                                                          "port": port, "host": hostname}
-                    except Exception:
-                        raise ValueError('Connection string \'%s\' is incorrect. Expected format for POSTGRESQL is '
-                                         'user:password@hostname:port/database' % connection_string_from_env)
-                    self.db_type = 'postgresql'
-
-                else:
```

Comment:
You have remove this 'else:' and the following 'else:' including the error messages: It is better to keep them in the code with the following message texts:
'The database type \'%s\' is unknown. Supported type is DB2' % db_type_from_env
'DB_TYPE is missing. Possible value for DB_TYPE is DB2'